src/lib_json/json_tool.h

```diff
@@ -19,6 +19,8 @@
 #include <clocale>
 #endif
 
+#include <algorithm>
```

Comment:
this is actually a heavy dependency to include just for a single (and simple) algorithm function to use

Reply:
I don't think it matters enough to optimize around it.
A header like this is likely to be brought in by other headers anyway.

Reply:
never directly and not but all compilers
(and it gets better each time)